shared/src/business/entities/trialSessions/TrialSession.test.js

```diff
@@ -429,13 +436,48 @@ describe('TrialSession entity', () => {
 
       expect(trialSession.canSetAsCalendared()).toBeFalsy();
     });
+
+    it('should be able to set a trial session as calendared if all properties are not empty for a remote session', () => {
+      const trialSession = new TrialSession(
+        {
+          ...VALID_TRIAL_SESSION,
+          chambersPhoneNumber: '111111',
+          joinPhoneNumber: '222222',
+          judge: { name: 'Judge Colvin' },
+          meetingId: '333333',
+          password: '4444444',
+          proceedingType: TRIAL_SESSION_PROCEEDING_TYPES.remote,
+        },
+        {
+          applicationContext,
+        },
+      );
+
+      expect(trialSession.canSetAsCalendared()).toBeTruthy();
+    });
+
+    it('should NOT be able to set a trial session as calendared if one or more properties are empty for a remote session', () => {
+      const trialSession = new TrialSession(
+        {
+          ...VALID_TRIAL_SESSION,
+          judge: {},
```

Comment:
Could we test more than the judge field here, since the judge field is always required and not just specifically for a remote session?

Reply:
In-person trial session support hasn't been added yet, because all sessions are currently remote.